homework-submission/react-hyf/react-hyf-client/src/components/HomeworkWeek3/Profile.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Gender from './Gender';
+import Region from './Region';
+import FullName from './FullName';
+
+const Profile = ({ gender, region, name, surname }) => {
+  return (
+    <div>
+      <table className={'profile-table'}>
+        <tbody>
+          <tr className={'tr'}>
```

Comment:
`tr` as a classname on `tr` is not good practice, give it a better name; `profile-row-fullname`